apps/backend/src/applications/applications.service.ts

```diff
@@ -22,6 +24,26 @@ export class ApplicationsService {
     return apps;
   }
 
+  async findAllCurrentApplications(): Promise<GetApplicationResponseDTO[]> {
+    const currentCycle: Cycle = getCurrentCycle();
+    const applications = await this.applicationsRepository.find({
+      where: {
+        //TODO q: I had to change Cycle definition to make year and semester public. Is there a reason it was private?
```

Comment:
⛏️ Rather than directly accessing the fields in `currentCycle`, we should make 2 new environment variables that hold the current year and current semester. We will have to do this eventually anyway. We can call them `NX_CURRENT_SEMESTER` and `NX_CURRENT_YEAR` in the `.env` file and access them here as `process.env.NX_CURRENT_SEMESTER` and `process.env.NX_CURRENT_YEAR.`

Reply:
now using .env variables